### PR TITLE
added crumbles

### LINKED ISSUE #57: Add Breadcrumbs
image

- Add breadcrumbs similar to the form application page, should be `Home > [Program Name]`

### PR DESCRIPTION
**Pull request by Shadow709**
image

### CODE REVIEW

File: src/app/admin/[program]/page.tsx [hunk 1]
```diff
@@ -26,6 +33,30 @@ const Page = () => {
...
+                <BreadcrumbItem>
+                  <BreadcrumbLink href="/">Home</BreadcrumbLink>
+                </BreadcrumbItem>
+              </BreadcrumbSeparator>
+
+              <BreadcrumbSeparator />
+              <BreadcrumbItem>
+                <BreadcrumbLink href="/form">Forms</BreadcrumbLink>
+              </BreadcrumbItem>
+              <BreadcrumbSeparator />
```
Comment (seanquiambao) [inline src/app/admin/[program]/page.tsx]: should be a program name like ignite or create, not forms.

Comment (seanquiambao) [inline src/app/admin/[program]/page.tsx]: image

remove excess separator

**Review by seanquiambao — CHANGES_REQUESTED** [review]

File: src/app/admin/[program]/page.tsx (hunk 1 above)
Comment (Shadow709) [reply src/app/admin/[program]/page.tsx]: image

File: src/app/admin/[program]/page.tsx [hunk 2]
```diff
@@ -26,6 +33,29 @@ const Page = () => {
...
+            <BreadcrumbList>
+              <BreadcrumbSeparator>
+                <BreadcrumbItem>
+                  <BreadcrumbLink href="/">Home</BreadcrumbLink>
+                </BreadcrumbItem>
+              </BreadcrumbSeparator>
+
+              <BreadcrumbSeparator />
+              <BreadcrumbItem>
+                <BreadcrumbLink href="/form">Ignite</BreadcrumbLink>
```
Comment (shahdivyank) [inline src/app/admin/[program]/page.tsx]: should not point to form

File: src/app/admin/[program]/page.tsx [hunk 3]
```diff
@@ -26,6 +33,29 @@ const Page = () => {
...
+              </BreadcrumbItem>
+
+              <BreadcrumbItem>
+                <BreadcrumbLink
+                  href={`/form/`}
```
Comment (shahdivyank) [inline src/app/admin/[program]/page.tsx]: should not point to form

File: src/app/admin/[program]/page.tsx [hunk 4]
```diff
@@ -26,6 +33,29 @@ const Page = () => {
...
+                  className="capitalize"
+                ></BreadcrumbLink>
```
Comment (shahdivyank) [inline src/app/admin/[program]/page.tsx]: should contain text, check the issue requirements

File: src/app/form/[role]/page.tsx [hunk 5]
```diff
@@ -28,14 +28,18 @@ const Page = ({ params }: props) => {
       <div className="w-1/2">
         <Breadcrumb>
           <BreadcrumbList>
-            <BreadcrumbItem>
-              <BreadcrumbLink href="/">Home</BreadcrumbLink>
-            </BreadcrumbItem>
+            <BreadcrumbSeparator>
```
Comment (shahdivyank) [inline src/app/form/[role]/page.tsx:31]: breadcrumbs should only be applied to one page, check the issue requirements

**Pull request closed without merging**